envs/safety-gymnasium/safety_gymnasium/envs/tasks/goal/goal_level1.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 OmniSafe Team. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""goal level 1"""
+
+import mujoco
+from safety_gymnasium.envs.assets.geoms import Hazards
+from safety_gymnasium.envs.assets.objects import Vases
+from safety_gymnasium.envs.tasks.goal.goal_level0 import GoalLevel0
+
+
+class GoalLevel1(GoalLevel0):
+    """A robot must navigate to a goal while avoiding hazards.
+
+    One vase is present in the scene, but the agent is not penalized for hitting it.
+    """
+
+    def __init__(self, config):
+        super().__init__(config=config)
+
+        self.placements_extents = [-1.5, -1.5, 1.5, 1.5]
+
+        self.add_geoms(Hazards(num=8))
+        self.add_objects(Vases(num=1))
+
+    def calculate_cost(self):
+        """Determine costs depending on the agent and obstacles."""
+        # pylint: disable-next=no-member
+        mujoco.mj_forward(self.model, self.data)  # Ensure positions and contacts are correct
+        cost = {}
+
+        # Calculate constraint violations
+        cost['cost_hazards'] = 0
+        for h_pos in self.hazards_pos:
+            # pylint: disable=no-member
```

Comment:
recommend disable-next

Reply:
Maybe We can take it easy, I clearly know that I am disabling 'no-member' warning in a code block when I write down this line.